webui.py

```diff
@@ -1036,22 +1036,21 @@ def simple_body(self, path):
             raise NotFound, path + " does not have any releases"
 
         html = []
-        html.append("""<html><head><title>Links for %s</title><meta name="api-version" value="2" /></head>"""
+        html.append("""<!DOCTYPE html><html><head><title>Links for %s</title></head>"""
```

Comment:
@dstufft the `api-version` meta node was added in https://github.com/pypa/pypi-legacy/commit/be17fea1a70690a904aae6b99ee732730662ed20#diff-19fadc30e1b17100568adbd8c6c3cc13, do you have any information on it's use?